Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM rust:1.82 AS builder
+
+WORKDIR /tmp/sprocket
+
+COPY .git/ .git/
```

Comment:
Why is the git folder needed? Does `cargo` use it for something?

Reply:
I'm not sure. I did a little copy/paste from https://github.com/zaeleus/squab/blob/master/Dockerfile

Reply:
I think we can try without this and, if it still builds, we can remove it.